doc/source/release-notes/Percona-Server-5.6.34-79.1.rst

```diff
@@ -0,0 +1,44 @@
+.. rn:: 5.6.34-79.1
+
+============================
+|Percona Server| 5.6.34-79.1
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.34-79.1 on
+November 23rd, 2016 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.34-79.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.34
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-34.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.34-79.1 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.34-79.1
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.34-79.1>`_.
+
+Deprecated Features
+===================
+
+ :ref:`scalability_metrics_plugin` feature is being built by default but
+ deprecated. Users who have installed this plugin but are not using its
+ capability are advised to uninstall the plugin due to known crashing bugs.
+
+Bugs Fixed
+==========
+
+ When a stored routine would call an "administrative" command such as
```

Comment:
Official MySQL docs talk about administrative commands without quotes, we should follow that convention